packages/v4/src/content/design-guidelines/components/modal/modal.md

```diff
@@ -7,111 +7,176 @@ related: ['Button']
 import CheckIcon from '@patternfly/react-icons/dist/esm/icons/check-icon';
 import TimesIcon from '@patternfly/react-icons/dist/esm/icons/times-icon';
 
-A **modal** loads and displays important information without navigating away from the current page.
+A **modal** displays important information to a user without requiring them to navigate to a new page.
 
 ## Primary elements
-<img src="./img/modal.png" alt="Modal" />
+
+<img src="./img/modal-call-outs.png" alt="An example of a modal in context with call outs to highlight each listed element" width="756"/>
 
 1. **Backdrop**
 2. **Modal box**
-3. **Title:** Concisely convey the purpose of the modal.
-4. **Close**
+3. **Headline**
+4. **Close icon**
 5. **Content area**
-6. **Buttons:** Enable a user to confirm, submit, or exit. Modal dialogs can include multiple action buttons, but should be limited to one primary button
+6. **Buttons** 
+
+See the [Content](#content) section for information about writing modal copy.
+
+## Spacing
+
+### Compact (24px)
+
+<img src="./img/modal-compact.png" alt="A compact modal with 24px spacers" width="487" />
+
+
+Compact modals are best used to display small amounts of content. 
+
+Their primary spacing of 24px separates:
+
+- Each edge of the modal box from all modal content.
+- The content area from both buttons.
+
+A 16px spacing separates the modal's headline from the content area. 16px also separates each button.
+
+### Large (32 px)
```

Comment:
Just realized that there actually is no longer a compact vs large modal. Now there's just a basic modal that has the compact spacing. So I would just say "Spacing" delete the "Compact" header, and call them "Basic modals" in the description.

Also if you could change the image to the one I sent you on slack - sorry that's my bad!